backend/utils/natural_language_processing_api.py

```diff
@@ -0,0 +1,44 @@
+from dotenv import load_dotenv
+import json
+from groq import Groq, AuthenticationError, APIConnectionError
+import os
+
+
+def prompt_nlp_model(prompt, api_inputted_key): 
+    """
+    Prompts the API and with information
+
+    Args:
+        str: A question or command for the model to fulfill
```

Comment:
it takes 2 parameters but only one param is documented. However, if you plan  to get rid of the api_key parameter, this wont be a problem anymore